docs/sequence-diagrams/README.md

```diff
@@ -0,0 +1,10 @@
+# Sequence Diagrams
```

Comment:
TBH I would say we would love to:
* Have those diagrams defined in some place
* Have automation (as part of `make docs?`) that will generate png those from `.puml` files.
* png will be used in `docs/components` for each component. (: What do you think?